ompi/mca/fs/lustre/fs_lustre_file_open.c

```diff
@@ -76,16 +76,30 @@ mca_fs_lustre_file_open (struct ompi_communicator_t *comm,
     perm = mca_fs_base_get_file_perm(fh);
     amode = mca_fs_base_get_file_amode(fh->f_rank, access_mode);
 
-    opal_info_get (info, "stripe_size", MPI_MAX_INFO_VAL, char_stripe, &flag);
+    opal_info_get (info, "striping_unit", MPI_MAX_INFO_VAL, char_stripe, &flag);
```

Comment:
Do we need to support the old info key names for backwards compatibility?  Or were those names erroneous somehow?

Reply:
we have the same logic in main and v5.0.x, so I think it would make sense to keep the same logic in v4.1.x as well

Reply:
The question still stands, though -- do we expect that there are any codes out there using the old info key names such that we need to keep supporting that functionality, too?  (in main, v4.1.x, and v5.0.x)

Reply:
I do not have any data to conclusively argue in one or the other direction. But when we introduced the name change of the info objects to match what is in the MPI spec, we decided to keep the old names just to not cause trouble to applications that potentially used the old names. Basically, we did not see a downside of keeping the old names as well, at least for a while.

Reply:
I mis-read the diff: I thought you were *removing* the old names.  Now I read the patch more carefully and I see you *added* new names and *kept* the old names.  That is exactly the Right Thing to do.

Sorry for the noise!